internal/operators/lvm/lvm_operator.go

```diff
@@ -179,3 +179,16 @@ func (o *operator) GetPreflightRequirements(context context.Context, cluster *co
 func (o *operator) GetSupportedArchitectures() []string {
 	return []string{common.X86CPUArchitecture, common.ARM64CPUArchitecture}
 }
+
+func (o *operator) getLvmMemoryPerHostMib(ctx context.Context, cluster *common.Cluster) int64 {
+	log := logutil.FromContext(ctx, o.log)
+	greaterOrEqual, err := common.BaseVersionGreaterOrEqual(cluster.OpenshiftVersion, "4.13.0")
```

Comment:
Maybe save that OCP version it as const? Similar that we did [here](https://github.com/openshift/assisted-service/blob/4a7d2a4b3b0fefd4fec2e0b7f6e50e972c217f88/internal/operators/lvm/config.go#L7)